docs/sql-ref-ansi-compliance.md

```diff
@@ -111,6 +111,13 @@ SELECT * FROM t;
 
 The behavior of some SQL functions can be different under ANSI mode (`spark.sql.ansi.enabled=true`).
   - `size`: This function returns null for null input under ANSI mode.
```

Comment:
(This is not related to this PR though) could you remove `under ANSI mode` in this statement, too?

Reply:
OK